langs/en.json

```diff
@@ -283,6 +283,18 @@
       "description": "Description is required"
     }
   },
+  "proposalComment": {
+    "modal-title": "Title: Proposal #{id}",
+    "modal-subtitle-against": "Please provide feedback about your decision. Why are you against of this proposal?",
+    "modal-subtitle-for": "Please provide feedback about your decision. Why are you for of this proposal?",
+    "form-contact": "Contact",
+    "form-contact-placeholder": "Enter contact (optional)",
+    "form-contact-tooltip": "Contact (optional) may be nickname in forum, email, telegram, twitter or others",
```

Comment:
Contact (optional) may be a nickname in forum, email, telegram, twitter or other

Reply:
Fix